api/RTCPeerConnection.json

```diff
@@ -1429,6 +1429,39 @@
             "standard_track": true,
             "deprecated": false
           }
+        },
+        "stopped_transceivers_removed": {
```

Comment:
FYI, I used the same key but different descriptions for each part - tranceivers, receiver, sender.